projects/plugins/wpcomsh/custom-colors/colors.php

```diff
@@ -1733,6 +1735,11 @@ public static function color_suggestions_from_palette( $colors, $role ) {
 				),
 			)
 		);
+
+		if ( empty( $top_palette ) ) {
```

Comment:
It'll always exist, so this is equivalent to just
```suggestion
		if ( ! $top_palette ) {
```